src/blessed-client/blessed-client.ts

```diff
@@ -41,7 +41,7 @@ export class BlessedClient {
    */
   private initiateBox() {
     // Create a box perfectly centered horizontally and vertically
-    const box = blessed.box({
+    return blessed.box({
```

Comment:
Is there a reason we're returning here instead of after setting up the keypress handlers? In my PR I modified this method to take an attributes parameter which changes this line to `const box = blessed.box(attributes)`

Reply:
o yeah no this is a problem